### DISCUSSION TITLE
Classification based on only spectral signatures

### DISCUSSION THREAD

**Discussion by 33-sl** [semiautomaticgit/SemiAutomaticClassificationPlugin#387] [Q&A] [answered]
Hello! I'm writing my master's thesis on alpine nature types, and have been using the Semi-autmatic classification plugin for making a classification. I collected field data this year, and mapped ROIs that I used as training input for my supervised classification. I want to see wether these nature types have changed through the last years, so I need to use only the signatures, and not the ROIs to do classifications of the same are of previous years. I have not quite figured out how to do this, as it seems the ROIs are necessary to do a supervised classification. Is there a way around this in this plugin where I can use only the signatures from my classes, not the ROIs? Thanks alot in advance!

**Comment by semiautomaticgit** [comment, marked as answer]
Hello @33-sl ,

you can export the spectral signatures as CSV file from the previous training input using the export tool
https://semiautomaticclassificationmanual.readthedocs.io/en/latest/export_signatures_tab.html

Then import these signatures in a new SCP training file using the import tool (after creating the new Band set)
https://semiautomaticclassificationmanual.readthedocs.io/en/latest/import_signatures_tab.html#import-library-file

Please note that only Spectral Angle Mapping and Maximum Likelihood algorithms work with spectral signatures (without ROIs).
